modules/yaml/yaml.go

```diff
@@ -100,29 +100,21 @@ func renderVerticalHtmlTable(m []yaml.MapSlice) string {
 	return table
 }
 
-func RenderYaml(data []byte) []byte {
+func RenderYaml(data []byte) ([]byte, error) {
 	mss := []yaml.MapSlice{}
 
 	if len(data) < 1 {
-		return data
-	}
-
-	lines := strings.Split(string(data), "\r\n")
-	if len(lines) == 1 {
-		lines = strings.Split(string(data), "\n")
-	}
-	if len(lines) < 1 {
-		return data
+		return data, nil
 	}
```

Comment:
Since you removed the splitting, you should include a test that verifies this works with Windows, Linux and Mac line endings.